### PR TITLE
Update `CWC 2022` Grand Finals schedule, mappool links

### PR DESCRIPTION
**Pull request by LeoFLT**
Adds the Grand Finals showmatch details, as well as updates DT1 and MM1's beatmapset links (previous diffs were deleted and reuploaded with fixes).

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/CWC/2022/en.md [hunk 1]
```diff
@@ -99,6 +99,12 @@ The osu!catch World Cup 2022 is run by the [osu! team](/wiki/People/The_Team) an
 
 ## Match schedule: Grand Finals
 
+### Saturday, 2 July 2022
+
+| Team A | Team B | Match time |  |
+| --: | :-- | :-- | :-: |
+| All-stars Red | All-stars Blue | [Jul 3 (Sun) 12:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220703T120000&p1=1440) | [^showmatch] |
```
Comment (cl8n) [inline wiki/Tournaments/CWC/2022/en.md:106]: seems like wrong date cuz overlaps with grand finals, and it doesn't match your header

Comment (LeoFLT) [reply wiki/Tournaments/CWC/2022/en.md]: I forgot to change the time for that table :/

Comment (cl8n) [reply wiki/Tournaments/CWC/2022/en.md]: time still doesnt match o_O

Comment (LeoFLT) [reply wiki/Tournaments/CWC/2022/en.md]: Fixed for good now

**Review by peppy — APPROVED** [review]

**Pull request merged**